pkg/config/constants.go

```diff
@@ -16,6 +16,7 @@ const (
 
 type images struct {
 	Pg15 string `mapstructure:"pg15"`
+	Pg17 string `mapstructure:"pg17"`
```

Comment:
```suggestion
	Pg string `mapstructure:"pg"`
```

Dependabot only updates one tag of the same image. So let's just rename it as pg.